### ISSUE TITLE
Allowing comma separated emails

### ISSUE THREAD

**Issue by almothafar** [wildbit/postmark-java#32]
It seems this issue started to happen recently since the 8th of November, it seems that contains breaking changes for some reason, I was using Message and TemplateMessage and sending emails to multiple addresses with a comma-separated form like `email1@company.com,email2@company.com`.

Currently, it starts to fire exceptions in the log:

```console
Caused by: com.wildbit.java.postmark.client.exception.InvalidMessageException: Error parsing 'To': Illegal email address 'email1@company.com,email2@company.com'. It must contain the '@' symbol.,
	at com.wildbit.java.postmark.client.HttpClientErrorHandler.throwErrorBasedOnStatusCode(HttpClientErrorHandler.java:37),
	at com.wildbit.java.postmark.client.HttpClientHandler.execute(HttpClientHandler.java:68),
	at com.wildbit.java.postmark.client.ApiClient.deliverMessageWithTemplate(ApiClient.java:167),
	at providers.emails.PostmarkEmailSenderServiceImp.sendMailWithTemplateTask(PostmarkEmailSenderServiceImp.java:170),
	at providers.emails.PostmarkEmailSenderServiceImp.lambda$sendMailWithTemplate$1(PostmarkEmailSenderServiceImp.java:69),
	at providers.emails.PostmarkEmailSenderServiceImp.lambda$sendMailScheduler$5(PostmarkEmailSenderServiceImp.java:112)

```

so, I wanted to fix that and I see there is a function called `.setTo()` that accepts an array or a map in `Message` and accepts Map only in `TemplatedMessage` which I don't understand and is confusing:

* First why I don't have a constructor that accepts a list or a map, do I need to create an object with "to" to be anything then use `.setTo`?
* why the `.setTo` accept only a map in `TemplatedMessage`, why I don't have a function that accepts a list in `TemplatedMessage` like `Message`, as I have addresses and without names in my current situation.
* I do want to be allowed to send my own email comma separated like before.
* Using names as a key in Map does not make sense for me, what if I have 2 persons with the same name but different addresses, or I have 2 email addresses for my own like `Al-Mothafar Al-Hasan<almothafar@company.com>` and `Al-Mothafar Al-Hasan<almothafar@another-company.com>`.

**Comment by tomazy** [comment]
@almothafar there haven't been any changes to the Postmark's API and it should l be working as expected - Postmark allows up to 50 recipients as comma separated list in a string:
https://postmarkapp.com/developer/api/templates-api#email-with-template

It seems to me that there is something wrong with the string you're passing as the "To" field. It is not `email1@company.com,email2@company.com` but `"email1@company.com,email2@company.com"` (note the surrounding quotes) instead - could you check that?

You can also show us how you're constructing that message with multiple recipients.

**Comment by almothafar** [comment]
@tomazy I don't have any quotes around emails, also, I checked logs everything was working fine from 28th of Oct to 7th of Nov, we did 0 changes to the code, and we didn't do any deployment on the 7th of 8th of Nov. where the issue starts to happen, I did another deployment in 14th of Nov. but still having the same issue.

I have something like this:

```java
String from = "noreply@mycompany.com";
String to = "email1@company.com,email2@company.com";
Integer templateId = 123;
String subject = "Test subject";

final TemplatedMessage templatedMessage = new TemplatedMessage(from, to, templateId);
templatedMessage.setReplyTo(from);
templatedMessage.setTemplateModel(Json.toJson(templateData));
client.deliverMessageWithTemplate(templatedMessage);
```

**Comment by almothafar** [comment]
OK, finally I saw the issue, To was "Name <email1@company.com,email2@company.com>" when I debugged it inside the postmark code itself, I found the reason for this, but I'm still confused, it was working like that before and now not working, or the change that deployed in 14th Nov. that written in 11th of Nov. somehow deployed to production in 7th to 8th of Nov., is that myself traveled in the time? or again that "Name <email1@company.com,email2@company.com>" was working and now not.

I'm totally confused...

Anyway, I fixed this from my side.
